articles/introduction-to-scrum/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: engineering-education
+status: publish
+published: false
+url: /engineering-education/introduction-to-scrum/
+title: Introduction to Scrum
+description: In this article we will speak about the Scrum agile framework used for development purposes.
+author: benson-kariuki
+date: 2020-09-29T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-scrum/hero.jpg
+    alt: Introduction to scrum
+---
+### Introduction
+Scrum is an agile framework for developing products. The framework is mostly used for software development. Scrum has been in use since the early 1990s. It is a flexible and holistic approach to project management. Scrum focus on continuous improvement, learning, and adjustment to the changing market, user requirements, and technology.
+<!--more-->
+
+Typically, Scrum has a small team made of highly adaptive and flexible individuals. Agile is an iterative approach to project management. Agile helps a team deliver a product to the customer faster, in different iterations. Customer feedback is critical in every iteration. There are several agile frameworks, such as Kanban and Scrum. The article will cover:
+
+- [Introduction to Scrum](#Scrum-Overview)
+- [Scrum Roles](#Scrum-Team)
+- [Scrum Events](#Scrum-Events)
+- [Scrum Artifacts](#Scrum-Artifacts)
+- [Why Use Scrum?](#Why-Use-Scrum?)
+
+### Scrum Overview
+Scrum has a few rules which are simple to understand. Scrum is challenging to master because it requires a culture change, which is not easy for most organizations. Scrum involves a set of events, tools, and roles that work together to help teams structure and manage a project.
+
+It is used in different fields, including marketing, manufacturing, and software development. In this article, we will use examples in software development. Within software development, requirements, market, technology, and other external factors change quite often. Scrum embraces the changes as the product is built in a series of iterations called sprints.
+
+Big projects are broken into small pieces that are delivered in several Sprints making the project easier to manage. Risk and cost are also reduced, as it is easier to identify risks in the early development stages. Every Sprint is released to users, and feedback is collected. Speed to market is also increased. Scrum is all about continuous learning and adapting to changes. The team does not know everything about the project. Therefore, adaptation to changes is critical.
+
+The image below is a graphical summary of the Scrum framework. If your organization would use Scrum, where would your input be? After going through this article, I bet you will be able to answer the question.
+
+![Scrum Framework](/engineering-education/introduction-to-scrum/scrum_framework.jpg)
```

Comment:
i believe this image needs a source 🚀 
@Tsanguu

Reply:
I borrowed a few concepts from other related images and came up with an original one. I designed it.
@hectorkambow